core/modules/system/system.theme.inc

```diff
@@ -428,14 +428,17 @@ function theme_admin_page($variables) {
  */
 function theme_admin_block($variables) {
   $block = $variables['block'];
+  // Construct a class.
+  $class_suffix = end(explode('/', $variables['block']['path']));
```

Comment:
This line is causing test failures, because `end()` can only affect variables that are passed by reference. This may need to be split into two lines.

Reply:
pushing a new PR now